api/turing/service/router_deployment_service_test.go

```diff
@@ -84,9 +82,6 @@ func (msb *mockClusterServiceBuilder) NewEnricherService(
 		BaseService: &cluster.BaseService{
 			Name:      fmt.Sprintf("%s-enricher-%d", rv.Router.Name, rv.Version),
 			Namespace: project.Name,
-			Labels: map[string]string{
-				"env": envType,
-			},
```

Comment:
This should no longer be here the actual `clusterServiceBuilder` does not even use an argument `envType` to set the labels.